templates/ibmcloud-powervs-doc.md

```diff
@@ -0,0 +1,150 @@
+### Provision local boostrap management cluster:
```

Comment:
we have started looking at hosting a doc for the project and the layout can be found here: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/docs/book/src,

and this content can go there and here is how it looks like post-deployment: cluster-api-aws.sigs.k8s.io/ and also you can refer some of the content from the other provider